kvmagent/kvmagent/test/vmmeta_testsuite/sblk/test_sblk_prefix_rebase.py

```
@@ -0,0 +1,260 @@
+        iscsi_server = env.get_vm_metadata('self')
+        rsp = storage_device_utils.iscsi_login(iscsi_server.ip, "3260")
+        self.assertEqual(rsp.success, True, "iscsiadm login failed")
+        r, o = bash.bash_ro("ls /dev/disk/by-id | grep scsi|awk -F '-' '{print $2}'")
+        blockUuid = o.strip().replace(' ', '').replace('\n', '').replace('\r', '')
+        rsp = self.connect([blockUuid], [blockUuid], vgUuid, hostUuid, hostId, forceWipe=True)
+        self.assertEqual(True, rsp.success, rsp.error)
+
+    # -- sub-tests ---------------------------------------------------------
+
+    def _test_empty_prefix_raises(self):
+        with self.assertRaises(Exception):
```

Ruff (0.15.7)

[warning] Do not assert blind exception: `Exception`

(B017)